packages/api/cms-api/src/user-permissions/content-scope.service.ts

```diff
@@ -43,7 +43,19 @@ export class ContentScopeService {
     }
 
     async inferScopesFromExecutionContext(context: ExecutionContext): Promise<ContentScope[]> {
-        return [...new Set((await this.getScopesForPermissionCheck(context)).flat())];
```

Comment:
This never worked, because using a Set to get unique values only works for primitives, not objects.